README.md

```diff
@@ -45,9 +48,36 @@ If you havent already you will want to install vagrant details found [`https://w
 
 Using a terminal you should change directory to this project and run the following
 
-```vagrant up```
+```bash
+vagrant up
+```
 
 vagrant will begin to build your app using gunicorn rather than flask, once the build scripts stop providing feedback you
 can now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+### Run using docker
```

Comment:
I don't think your non-Docker instructions work anymore. You've made it so you need to run your poetry commands from the todo-app subdirectory.

And the .env file might also need to be in that subdirectory for `poetry run flask run` to pick it up.

Reply:
Hey Hugh, rather than including the .env I have added to .dockerignore and reference it in the run commands/compose file